luaj/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
You don't need the gradle wrapper in the luaj subproject.
You can just use the parent gradlew wapper to run the project.